temporalio/client/client.py

```diff
@@ -0,0 +1,131 @@
+from dataclasses import dataclass
+from datetime import timedelta
+from enum import Enum
+from typing import Any, Awaitable, Mapping, Optional, Union
+
+import temporalio
+import temporalio.client
+import temporalio.converter
+
+
+@dataclass
+class TLSConfig:
+    server_root_ca_cert: Optional[Union[str, bytes]] = None
+    domain: Optional[str] = None
+    client_cert: Optional[Union[str, bytes]] = None
+    client_private_key: Optional[Union[str, bytes]] = None
+
+
+class WorkflowIDReusePolicy(Enum):
+    ALLOW_DUPLICATE = 1
+    ALLOW_DUPLICATE_FAILED_ONLY = 2
+    REJECT_DUPLICATE = 3
```

Comment:
We can, but I think people should be able to use our high-level client wrapper using only types from our high-level client lib. Protos, especially since we don't commit them, are tough to read/understand all the types on. These of course will just convert to protos.

Reply:
Removed